data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+			`	Nasilor stops by a door immediately after one of these "viewing sites", and opens it. He introduces you to his sister, and you two start helping her pack what remains. Most of what they're bringing are several research papers, though you can't know what's written on them.`
+			`	Finally, you start heading back to your ship. Noticing your interest in taking in the beauty of the gypsum caves, the two Saryd researchers slow down their pace and start to tell you more about themselves. They are twins, both born on Saros, but have been working here for decades. Though you don't understand much of the technical jargon, you figure out that Nasilor has specialized in cryptocrystalline compounds, while Alituri focused her studies on planetary and cosmic conditions that lead to the formation of the gymsum caves here.`
+			`	With the little "tour" over, when you three reach your ship, you show them to their bunks, and prepare to head back to <planet>.`
+				accept
+
+
+
+mission "Coalition Yottrite 6"
+	name "Master Shipwright"
+	description "Head to <destination> to pick up a member of House Mallob, so that they may help in trying to apply Yottrite to an alloy."
+	landing
+	to offer
+		has "Coalition Yottrite 5: done"
+	source "Delve of Bloptab"
+	destination "Market of Gupta"
+	on offer
+		conversation
+			`Lugglop is waiting for you when you land, and greets the two researchers as you help them get their things out of the ship.`
+			`	He points them to a hotel, where he had a room prepared for them, and asks you to follow him to his office again.`
+			`	"Filled to the brim with cutting edge equipment, our facilities here and on the neighbor planet are. Alas, venture much into the theoretical field needed to study such rare elements, our personnel here does not."`
+			`	He shows you to his map again, this time pointing to a nearby Arachi world. "Once cracked down some aspects of Yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`
+			`	You're about to head back to your ship, when he hands you 200,000 credits. "Nearly forgot to pay you for this last service, I did."`
+				accept
+	on accept
+		payment 200000
+
+
+
+mission "Coalition Yottrite 7"
+	name "Master Shipwright"
+	description "Bring Pichiie to <destination>, where she will join Lugglop's team in their Yottrite research."
+	passengers 1
+	landing
+	to offer
+		has "Coalition Yottrite 6: done"
+	source "Market of Gupta"
+	destination "Delve of Bloptab"
+	on offer
+		conversation
+			`You're immediately hailed by House Mallob when your ship is descending the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive castle-like structure, which appears to be made out of a similar material to that of Arach ships, at the base of a mountain.`
```

Comment:
```suggestion
			`You're immediately hailed by House Mallob when your ship enters the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive, castle-like structure at the base of a mountain. It appears to be made out of a similar material to that of Arach ships.`
```
Long sentence (personal preference). Two adjectives describing `structure`, need a comma between them. Moved `at the base of a mountain` closer to what it describes for clarity.

Reply:
did my change help clarify, or does it need to be changed still?